packages/govuk-frontend/src/govuk/components/accordion/accordion.test.js

```diff
@@ -733,6 +733,45 @@ describe('/components/accordion', () => {
               message: 'GOV.UK Frontend is not supported in this browser'
             })
           })
+
+          it('throws when $module is not set', async () => {
+            await expect(
+              renderAndInitialise(page, 'accordion', {
+                params: examples.default,
+                beforeInitialisation() {
+                  // Remove the root of the components as a way
+                  // for the constructor to receive the wrong type for `$module`
+                  document.querySelector('[data-module]').remove()
+                }
```

Comment:
Mind if we try an keep the boilerplate down?

Hopefully obvious that `$module.remove()` throws `MissingElementError`

```suggestion
                beforeInitialisation: ($module) => $module.remove()
```

Reply:
Oooh, hadn't thought of that, yeah, that's shorter :D

Reply:
Single line arrow functions work too 👍